winpr/libwinpr/thread/process.c

```diff
@@ -478,6 +487,11 @@ BOOL TerminateProcess(HANDLE hProcess, UINT uExitCode)
 static BOOL ProcessHandleCloseHandle(HANDLE handle)
 {
 	WINPR_PROCESS* process = (WINPR_PROCESS*)handle;
+	if (process->fd >= 0)
```

Comment:
I think you should first check if the handle is valid?

Reply:
I can add an assert, but when we're here the handle has been dereferenced multiple time by caller code